docs/en/getting-started.md

```diff
@@ -79,3 +79,31 @@ Once you have a database connection instance, you can execute a **SQL** query by
 1. [Create a command with a plain SQL query](/docs/en/queries/create-command.md)
 2. [Bind parameters](/docs/en/queries/bind-parameters.md)
 3. [Call one of the SQL execute method to execute the command](/docs/en/queries/execute-command.md)
+
+
+## Quoting Table and Column Names
+
+When writing database-agnostic code, properly quoting table and column names is often a headache because different databases have different name quoting rules. To overcome this problem, you may use the following quoting syntax introduced by [Yii DB](https://github.com/yiisoft/db):
+
+- `[[column name]]`: enclose a column name to be quoted in double square brackets.
+- `{{table name}}`: enclose a table name to be quoted in double curly brackets.
```

Comment:
{{%table_name}}  - need recommended add percent before name for use prefix of tables

Reply:
and need use {{%table_name}} in all documentations. It's recommended example of usage